salt/states/file.py

```diff
@@ -3945,7 +3945,8 @@ def prepend(name,
             sources=None,
             source_hashes=None,
             defaults=None,
-            context=None):
+            context=None,
+            header=None):
```

Comment:
We need to add documentation to the docstring of this function to explain what this does, please.